bounties/npm/react-oauth-flow/1/vulnerability.json

```diff
@@ -0,0 +1,57 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Improper Authorization",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "react-oauth-flow",
+		"URL": "https://www.npmjs.com/package/react-oauth-flow",
+		"Downloads": "81421"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-255",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "L",
+		"PR": "N",
+		"UI": "N",
+		"S": "U",
+		"C": "H",
+		"I": "N",
+		"A": "N",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "7.5"
+	},
+	"CVEs": [
+		""
+	],
+	"Repository": {
+		"URL": "https://https://github.com/adambrgmn/react-oauth-flow",
+		"Codebase": [
+			"JavaScript"
+		]
+	},
+	"Permalinks": [],
+	"References": [
+		{
+			"Description": "GitHub Issue",
+			"URL": "https://github.com/ethereum/web3.js/issues/2739"
```

Comment:
This permalink seems totally unrelated to this issue. I don't know why npm has linked this in [their security advisory](https://www.npmjs.com/advisories/1487). This permalink is regarding a vulnerability that's already on our platform listed as a bounty: https://www.huntr.dev/bounties/1-npm-web3.